### PR TITLE
Fixing trsv rocblas template call

### PR DESCRIPTION
**Pull request by daineAMD**

### CODE REVIEW

File: library/src/include/rocblas.hpp [hunk 1]
```diff
@@ -1511,8 +1511,9 @@ rocblas_status rocblasCall_trsv(rocblas_handle handle,
                   "shiftA:", offset_A, "lda:", lda, "shiftB:", offset_B, "ldb:", ldb,
                   "bc:", batch_count);
 
+    // nullptr f or optional alpha
```
Comment (jzuniga-amd) [inline library/src/include/rocblas.hpp:1514]: f or -> for

**Review by jzuniga-amd — APPROVED** [review]
LGTM
Please just fix clang formatting and maybe wait for tests to pass. I guess once Andrew's PR is merged this one should pass all tests too. 
Thanks.

**Review by tfalders — APPROVED** [review]

**Comment by daineAMD** [comment]
I see https://github.com/ROCmSoftwarePlatform/rocBLAS-internal/commit/7b2ea658603f160b336b437676f25c0c6128a4b9 made some changes too, I'll update those here in a couple of minutes as well.

**Comment by tfalders** [comment]
The latest commit seems to be creating a merge conflict. I just merged #350, which is probably why.

**Comment by daineAMD** [comment]
My fault, I guess I missed that PR.

**Comment by jzuniga-amd** [comment]
@daineAMD tests seem to be passing now. Can we merge this PR? we need to unblock rocsolver CI asap

**Pull request merged**